pandas/io/tests/parse_dates_tests.py

```diff
@@ -0,0 +1,471 @@
+"""
+Tests date parsing functionality for all of the
```

Comment:
But then won't that file be run as a test suite?  I don't want that (it needs to be imported as a test class).